packages/style-guide/eslint/rules/next.js

```diff
@@ -10,5 +11,11 @@ export default [
       ...pluginNext.configs.recommended.rules,
       ...pluginNext.configs['core-web-vitals'].rules,
     },
+    languageOptions: {
+      globals: {
+        ...globals.browser,
+        ...globals.node,
```

Comment:
thought: `...globals.node,` should perhaps be set in `javascript.js` as it could be used anywhere, not only in next evironments.